pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorTest.java

```diff
@@ -805,7 +805,7 @@ public void testReplicatorOnPartitionedTopic(boolean isPartitionedTopic) throws
 
         log.info("--- Starting ReplicatorTest::{} --- ", methodName);
 
-        final String namespace = "pulsar/partitionedNs-" + isPartitionedTopic;
+        final String namespace = "pulsar/partitionedNs-" + isPartitionedTopic + "-" + System.nanoTime();;
```

Comment:
1）I searched the code, but I didn’t see a namespace with the same name in other places
2）There are two ";"

Reply:
fixed the duplicate ";"
It could be namespace left from other test run, the zookeeper used for test will have a temp dir to store data with same name. It could be really hard to debug in github action VM but make namespace unique across different test run should solve the issue for us.